CSharp/Library/Microsoft.Bot.Connector.Shared/OAuthAPI/OAuthApi.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.Bot.Connector
     /// <summary>
     /// OAuthClient operations.
     /// </summary>
-    public partial class OAuthApi : IServiceOperations<OAuthClient>, IOAuthApi
+    public partial class OAuthApi : IServiceOperations<OAuthClient>, IOAuthApiEx
```

Comment:
So the methods for the old IOAuthAPI implementation are there but we removed it from line 26, so the OAuthAPI class does not implement the interface anymore. could any backward compat issues stem from that? i.e. from user written code that used OAuthAPI and depended on the fact that it implemented IOAuthAPI interface? Even though it sounds unlikely, if it is a backward compat violation we must fix it. Seems like that case could be mitigated by simply leaving this as :

`public partial class OAuthApi : IServiceOperations<OAuthClient>, IOAuthApiEx, IOAuthApi`

Thoughts?